src/engraving/libmscore/textlinebase.cpp

```diff
@@ -466,7 +466,7 @@ void TextLineBaseSegment::spatiumChanged(qreal ov, qreal nv)
     _endText->spatiumChanged(ov, nv);
 }
 
-static constexpr std::array<Pid, 26> pids = { {
+static constexpr std::array<Pid, 26> TextLineBasePropertyId = { {
```

Comment:
Part of #8798

Reply:
True. It was needed to make this PR work. I did exactly the same as in that PR, so that it can be auto-merged and no conflict will occur (hopefully).

Reply:
I see